Backend/app/dashboard/CommonAPIs/spec/RiderPlatform/Management/API/Customer.yaml

```diff
@@ -18,12 +18,15 @@ apis:
       response:
         type: CustomerListRes
   - DELETE:
-        endpoint: /delete
-        auth: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS CUSTOMER_DELETE
-        params:
-          customerId: Id Customer
-        response:
-          type: APISuccess
+      endpoint:  /{customerId}/delete
+      auth: ApiAuth APP_BACKEND_MANAGEMENT CUSTOMERS CUSTOMER_DELETE
+      params:
+        customerId: Id Customer
+      response:
+        type: APISuccess
+      migrate:
+        endpoint: CustomerAPI DeleteCustomerEndpoint
+        endpointV2:
```

Comment:
`endpointV2` not required, only `endpoint`
`endpointV2` format does not used anywhere now, so maybe I'll remove this migration at all from DSL